app/views/home/about.html.haml

```diff
@@ -14,8 +14,47 @@
     semua orang. Selain itu, kontes ini berisi soal-soal berkualitas
     yang sangat cocok untuk persiapan OSN tingkat kabupaten, provinsi,
     maupun nasional.
+
   %h2 Tim Kami
-  .row#daftar-panitia
-  \... dan banyak lainnya!
-  %h3 Alumni Tim Kami
-  .row#alumni-panitia
+  .row
+    - endFlag = true
+    - @team.each_with_index do |data, index|
```

Comment:
Can we abstract these complex logic into `app/helpers` folder? You can check the functions there to get an idea of how stuff is organised there.

Ideally we want our views to be rather simple with all the view-related logic to be tucked away there.

Reply:
Yup sure, thanks for letting me know!